epicure-admin/src/app/components/generic-table/generic-table.component.scss

```diff
@@ -0,0 +1,63 @@
+.table-container{
+  margin: 20px;
+  border: 1px solid #cccccc;
```

Comment:
do you have a colors data file? put the color there

Reply:
added color file

Reply:
move the color there and use it here instead of hardcoded

Reply:
done